### PR TITLE
docs: error on docstrings for launcher methods

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says

### CODE REVIEW

File: src/ansys/geometry/core/connection/launcher.py [hunk 1]
```diff
@@ -380,10 +380,11 @@ def launch_modeler_with_discovery(
...
         a SystemError will be raised.
-        Possible values:
-        Ansys products Versions and their corresponding int values:
-        23.2 -> value : 232
-        24.1 -> value : 241
+
+        **Ansys products versions and their corresponding int values:**
+
+        * 23.2 -> value : ``232``
+        * 24.1 -> value : ``241``
```
Comment (AlejandroFernandezLuces) [inline src/ansys/geometry/core/connection/launcher.py:387]: It's kind of obvious that it is the int equivalent right? Also, to be uniform with the other lists of equivalences I would put first the int value and then what it represents.
```suggestion
        * ``232`` : 23.2
        * ``241`` : 24.1
```

File: src/ansys/geometry/core/connection/launcher.py [hunk 2]
```diff
@@ -472,10 +474,11 @@ def launch_modeler_with_spaceclaim(
...
```
Comment (AlejandroFernandezLuces) [inline src/ansys/geometry/core/connection/launcher.py:481]:
```suggestion
        * ``232`` : 23.2
        * ``241`` : 24.1
```

**Review by AlejandroFernandezLuces — DISMISSED** [review]
LGTM, a minor comment you might want to consider.

**Review by AlejandroFernandezLuces — DISMISSED** [review]

**Comment by RobPasMue** [comment]
Need re-approval

**Review by AlejandroFernandezLuces — APPROVED** [review]

**Pull request merged**